pills/01-why-you-should-give-it-try.xml

```diff
@@ -122,89 +121,89 @@
    </section>
 
    <section>
-     <title> Being pure functional</title>
+     <title> Being purely functional</title>
 
      <para>
        Nix makes no assumptions about the global state of the system.
        This has many advantages, but also some drawbacks of course.
-       The core of a Nix based system is the Nix store, usually
-       installed under /nix/store, and some tools to manipulate the
-       store. In Nix there's the notion of a derivation rather than a
-       package. The difference might be subtle at the beginning, so I
-       will often mix both of the words, ignore that.
+       The core of a Nix system is the Nix store, usually
+       installed under <literal>/nix/store</literal>, and some tools to manipulate the
```

Comment:
If you want to go Ultimate Docbookery you could use `<filename>` or even `<filename class="directory">`, but this is by no means necessary :)